outputs.tf

```diff
@@ -1,11 +1,7 @@
-output "id" {
-  value = aws_secretsmanager_secret.default.id
-}
-
 output "arn" {
   value = aws_secretsmanager_secret.default.arn
 }
 
 output "rotation_enabled" {
-  value = aws_secretsmanager_secret.default.rotation_enabled
+  value = aws_secretsmanager_secret_rotation.default.rotation_enabled
```

Comment:
This change was necessitated by the deprecation of the `rotation_enabled` attribute on the `aws_secretsmanager_secret` resource. I'm not sure why we're bothering to output this at all, but the value of `rotation_enabled` is now retrieved from the `aws_secretsmanager_secret_rotation` object instead of the original.